### ISSUE TITLE
Investigate making Expansion Panel

### ISSUE THREAD

**Issue by traviskaufman** [material-components/material-components-web#46]
Spec reference: https://www.google.com/design/spec/components/expansion-panels.html

- [ ] Single Expansion Panel
- [ ] Expansion panel group / accordion (a la Inbox)

> Copied from https://github.com/google/material-design-lite/issues/4499

**Comment by gurmukhp** [comment]
This panel would be super useful for a project we're working on.

**Comment by Garbee** [comment]
Some quick thoughts on this, `<details>` could be used to natively get the collapse bit done without JS in supported engines. And we could recommend [the polyfill](https://github.com/rstacruz/details-polyfill) for people who need support in older engines. With a little bit of styling on the `<summary>` and a node or two for the inner contents and actions.

**Comment by Garbee** [comment]
FTR a [functioning example](http://codepen.io/Garbee/pen/egjxBv) of what I'm thinking. Looking to add this into MDL first to try and get an early gauge on problems with the implementation for any concerns that should be addressed for MCW going forward.

**Comment by MCGPPeters** [comment]
Hi guys.. I've build an expansion panel according to the MD specs as an [aurelia component](https://github.com/MCGPPeters/expansion-panel/) , that you are welcome to use as a starting point for instance. I needed it for a project so I built one that works for our purposed. I'm not a dedicated front end developer, so you will be most likely be able to improve it. 

Cheers

**Comment by rudza** [comment]
Any progress on this?

**Comment by ColdRevsEnt** [comment]
Would love to see this implemented soon!

**Comment by stefanoguglielmi** [comment]
come on guys!!!

**Comment by touficbatache** [comment]
When is it coming? It definitely is what I need for my project!

**Comment by hebbet** [comment]
![image](https://user-images.githubusercontent.com/1423115/27791839-48734db6-5ff7-11e7-9569-fcc8fa9d8797.png)
according to https://www.pivotaltracker.com/n/projects/1664011

**Comment by touficbatache** [comment]
😑😒

**Comment by aecz** [comment]
![image](https://user-images.githubusercontent.com/7781120/32169886-85f0a14c-bd72-11e7-9e55-968d24e7e05e.png)
Pushed back to end of January 2018. I will use Angular Material one for now.

**Comment by putuoka** [comment]
Rip me. Already use MDC for 80% of my design till i realize it have no date picker component. what should i do now?

**Comment by griest024** [comment]
I've started a WIP for this component. See #2020

[Engineering Outline](https://docs.google.com/document/d/1y0092LZJABV-T2_dKX4THlwb7l9j8-zg3yh5J4uWTko/edit?usp=sharing)

**Comment by touficbatache** [comment]
@griest024 Thank you so very much. This component is very needed in my projects.

**Comment by griest024** [comment]
@touficbatache Yep me too. I have needed this for a while.

**Comment by touficbatache** [comment]
@griest024 I would have created the component and pushed it into MDC but I'm not sure how their system work and I don't have time..

**Comment by kfranqueiro** [comment]
This is no longer within scope of the MD spec, so we're no longer planning to add this to the core MDC Web repository.

**Comment by clevercodenl** [comment]
> This is no longer within scope of the MD spec, so we're no longer planning to add this to the core MDC Web repository.

I've found the collapsable/expandable for list still in the [MD spec](https://material.io/components/lists#types) at the **Expand and collapse**   section... Or am I looking at the wrong spec ?

**Comment by asyncLiz** [comment]
The expansion panel _as a standalone component_ is not _**currently**_ within spec at the time of writing.

Check out https://github.com/material-components/material-components-web/issues/5939#issuecomment-634336770 for a longer description of the hows and whys 🙂

I'm going to lock this thread since this is nearly a 4 year old issue with inactivity for 2 years. #5939 has the latest information from Material Design on expansion panels.

**tl;dr** we know they're hotly requested and we've made that known to our designers so that they can prioritize it.
